proposal-goto-operator.md

```diff
@@ -0,0 +1,50 @@
+## motivation
+Позволяет переместить выполнение кода в новое,  желательно, неожиданное место
+
+> Чрезмерное употребление ЭТОГО оператора может придать коду вид известного блюда итальянской кухни
+
+## high-level api
+### global scope
+
+```javascript
+code1();
+goto foo;
+code2();
+foo:
+code(3);
+
+// сработает так
+code1();
+code3();
+```
+
+### working in class-scopes
+
+```javascript
+class Foo {
+  bar() {
+    code1()
+    goto foo;
+  }
+
+  baz() {
+    code2()
+    foo:
+    code3()
+  }
+
+// usage
+const foo = new Foo();
+foo.bar();
+
+// сработает так
+code1();
+code3(); 
+// перенос в функциях происходит со всем скоупом - так веселее.
+```
+
+## FAQ
```

Comment:
## FAQ
### Why we need this?
1) So that experienced developers can more easily switch to JS from their obsolete languages.
2) So that new  developers taste life.
3) interviewers will be able to humiliate accidents at interviews in new ways.